plugins/check-path/core-rs/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@player-ui/check-path-plugin-rs",
+  "main": "index.js",
+  "module": "index.js",
+  "dependencies": {
+    "@player-ui/asset-transform-plugin": "^0.3.0",
+    "@player-ui/check-path-plugin": "^0.3.0",
+    "@player-ui/make-flow": "^0.3.0",
+    "@player-ui/player": "^0.3.0"
+  }
+}
```

Comment:
These should probably be linked to the local packages similar to how they are pulled in for other inter-project dependencies.

Reply:
good catch, will move these over to the `BUILD` file.